ansible/roles/test/files/ptftests/fast-reboot.py

```diff
@@ -204,29 +377,35 @@ def get_mac(self, iff):
 
     def generate_from_t1(self):
         self.from_t1 = []
-        self.ip_addr = []
 
         vlan_ip_range = self.test_params['vlan_ip_range']
 
         _, mask = vlan_ip_range.split('/')
         n_hosts = min(2**(32 - int(mask)) - 3, self.max_nr_vl_pkts)
```

Comment:
My code calculates the minimum number of hosts which could be used with current size of Vlan network. Our preference is to check against 1000 vlan hosts. But if size of vlan network is less then /22, we can't test for 1000 vlan hosts. So I take maximum possible value for the vlan network size..